dev/com.ibm.ws.transport.http/resources/OSGI-INF/welcome/nls/messages.js

```diff
@@ -0,0 +1,36 @@
+var messages = {
+    WELCOME_OPEN_LIBERTY: "Welcome to Open Liberty",
+    OPEN_LIBERTY_APP_SERVER: "Open Liberty Application Server",
+    WEBSPHERE_OPEN_LIBERTY_LOGO: "WebSphere Liberty Logo",
+    WELCOME_LIBERTY_SECTION: "Welcome to ",
+    WELCOME_IBM_OPEN_SOURCE: "An IBM Open Source Project",
+    RESOURCES: "Resources",
+    GROUPS_IO_FORUM: "Groups.io forum",
+    COPYRIGHT_OPEN_LIBERTY: "Licensed Materials — Property of the Open Liberty project, Copyright IBM Corp and others 1997, 2019. Project is distributed under the terms of the Eclipse Public License v1.",
+    WELCOME: "Welcome to Liberty",
+    WEBSPHERE_LIBERTY_LOGO: "WebSphere Liberty Logo",
+    WEBSPHERE_OPEN_LIBERTY_FOOTER: "WebSphere Liberty footer Logo",
+    WELCOME_SECTION: "Welcome Section",
+    WELCOME_SECTION_CONTENT: "Welcome Section Content",
+    WEBSPHERE_APPSERVER: "WebSphere Application Server Beta",
+    HOW_TO_SETUP_ADMIN_CENTER: "How to Setup Admin Center",
+    OPEN_ADMIN_CENTER: "Open Admin Center",
+    ADDITIONAL_RESOURCES: "Additional Resources",
+    EXTERNAL_RESOURCES: "External Resources",
+    WASDEV: "WASdev Community",
+    LIBERTY_DOC: "Liberty Documentation",
+    WASDEV_FORUM: "WASdev Forum",
+    HEADER_UPDATE_AVAILABLE: "Update Available: ",
+    HEADER_DOWNLOAD_LINK: "Click to download {0}", // Product name
+    HEADER_UPDATE_AVAILABLE_DOWNLOAD_LINK: "Update Available. Click to download {0}", // Product name
+    COPYRIGHT: "Copyright",
+    IBM_LOGO: "IBM Logo",
+    IBM_LICENSED: "Licensed Materials — Property of IBM © Copyright IBM Corp. 1997, 2019. All Rights Reserved. IBM, and the IBM logo are trademarks or registered trademarks of International Business Machines Corp., registered in many jurisdictions worldwide. Other product and service names might be trademarks of IBM or other companies. A current list of IBM trademarks is available on the Web at {0} ", // COPYRIGHT_URL
```

Comment:
Is it nescesary to have this IBM copyright statement in Open Liberty code? If possible it should be removed.